webanno/webanno
# #1494 - Merge/collapse relationship arcs

**What's in the PR**
- Allow enabling the arc-collapsing feature of the brat visualization

**How to test manually**
* Enable "collapse arcs" in the preferences dialog on the annotation page
* Create a document with 3 sentences
* Create multiple relations from the first to the last sentence
* See how the relations collapse in the second sentence

**Automatic testing**

**Documentation**
